website/content/en/docs/getting-started/users.md

```diff
@@ -115,4 +115,25 @@ To deploy ClusterLink on another cluster, please repeat steps 2-3 in a console w
 ## Try it out
 
 Check out the [ClusterLink Tutorials]({{< ref "tutorials" >}}) for setting up
- multi-cluster connectivity for applications using two or more clusters.
+multi-cluster connectivity for applications using two or more clusters.
+
+## Uninstall ClusterLink
+
+To uninstall the ClusterLink CLI, use the following command:
+
+{{< tabpane text=true >}}
+{{% tab header="Regular User" %}}
+
```

Comment:
both root and non-root can be combined with 
```sh
$ rm `which clusterlink`
```

Reply:
also, not that the installation steps did not differentiate between `root` and non-`root` users so it's a bit surprising to make the distinction on uninstall, IMHO